http4s-server/src/main/scala/com/avast/server/toolkit/http4s/Http4sServerConfig.scala

```diff
@@ -0,0 +1,37 @@
+package com.avast.server.toolkit.http4s
+
+import java.util.concurrent.TimeUnit
+
+import com.avast.server.toolkit.http4s.Http4sServerConfig.SocketOptions
+import org.http4s.blaze.channel
+import org.http4s.server.defaults
+
+import scala.concurrent.duration.{Duration, FiniteDuration}
+
+final case class Http4sServerConfig(
+  listenAddress: String = "127.0.0.1",
+  listenPort: Int = 8080,
+  nio2Enabled: Boolean = true,
+  webSocketsEnabled: Boolean = false,
+  http2Enabled: Boolean = false,
+  responseHeaderTimeout: FiniteDuration = Duration(defaults.ResponseTimeout.toNanos, TimeUnit.NANOSECONDS),
+  idleTimeout: FiniteDuration = Duration(defaults.IdleTimeout.toNanos, TimeUnit.NANOSECONDS),
+  bufferSize: Int = 64 * 1024,
```

Comment:
Are these constants the same as in the underlying classes? If so then we could have here `Option[Int]` 🤔

Reply:
Yes, the types and values are taken from `http4s-blaze-server`. If you mean to make it an `Option` in order to let the default values be defined only in `http4s-blaze-server`, that's a good idea but than everything would be `Option` in our case class and that wouldn't be very nice, don't you think?